libs/designer/src/lib/core/actions/bjsworkflow/initialize.ts

```diff
@@ -473,24 +472,18 @@ export const initializeCustomCodeDataInInputs = (inputs: NodeInputs, nodeId: str
   }
 };
 
-export const updateCustomCodeInInputs = async (
-  nodeId: string,
-  fileExtension: string,
-  nodeInputs: NodeInputs,
-  customCode: CustomCodeFileNameMapping
-) => {
+export const updateCustomCodeInInputs = async (parameter: ParameterInfo, customCode: CustomCodeFileNameMapping) => {
   if (isNullOrEmpty(customCode)) {
     return;
   }
-  // getCustomCodeFileName does not return the file extension because the editor view model is not populated yet
-  const fileName = getCustomCodeFileName(nodeId, nodeInputs) + fileExtension;
+  const language: EditorLanguage = parameter?.editorOptions?.language;
```

Comment:
Is the language always defined or do we need some default language as well?

Reply:
language should be defined in the manifest. Before this function in an earlier function we check if customCodeParameter.editorOptions.language is one of the languages supported. And if somehow the manifest gives us an undefined language, it won't ever get to this part of the code